tools/crm_verify.c

```diff
@@ -260,14 +260,14 @@ main(int argc, char **argv)
         const char *verbose_hint = "";
 
         if (pcmk__config_has_error) {
-            failure_type = " (with errors)";
+            failure_type = "invalid (with errors)";
         } else if (pcmk__config_has_warning) {
-            failure_type = " (with warnings)";
+            failure_type = "may need attention (with warnings)";
         }
         if (options.verbosity == 0) {
             verbose_hint = " (-V may provide more detail)";
         }
-        out->err(out, "Configuration invalid%s%s", failure_type, verbose_hint);
+        out->err(out, "Configuration %s%s", failure_type, verbose_hint);
```

Comment:
Looking at the complete messages in our regression tests, I don't think "(with errors)" and "(with warnings)" adds anything useful, so you can just get rid of those.  Also, please run `cts/cts-cli -r crm_verify -s` and include those changes in your patch.